### DISCUSSION TITLE
[Feature] Show comments from image metadata

### DISCUSSION THREAD

**Discussion by roasted-watermelon** [immich-app/immich#10707] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

We can set comments in metadata using exiftool as below
```
exiftool -UserComment="Original date inferred from modification date" -m -G0 -overwrite_original -P $file
```

It shows up in gthumb as such:  

<img src=https://github.com/immich-app/immich/assets/156080803/c95cc407-b149-430b-a331-1bd1ba49b23c width=350/>

My particular use case is I have many photos from really old mobile phones which did not set an `DateTimeOriginal` so the modification date is the closest I can get. So I set a warning in the image with the comment.

It would be great if we could see this comment directly on the web/mobile without having to download the image.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#6237

**Reply by roasted-watermelon** [reply]
@bo0tzz thank you.
